frontend/src/components/Toolbar/Toolbar.tsx

```diff
@@ -3,43 +3,50 @@
  * @author jhuebner
  * @date 06.09.2021
  */
+
 import React, { useState } from 'react';
+import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { FaBars } from 'react-icons/fa';
 import Logo from '../../images/kant.png';
 import useWindowDimensions from '../../hooks/useWindowDimensions';
 import './Toolbar.css';
 import { LoginLogoutButton } from './LoginLogoutButton/LoginLogoutButton';
 
-const links = [
-  {
-    id: 1,
-    name: 'AG-Buchung',
-    path: '/student/ag',
-  },
-  {
-    id: 2,
-    name: 'Facharbeit',
-    path: '/student/facharbeit',
-  },
-  {
-    id: 3,
-    name: 'Wahlpflicht',
-    path: '/student/wahlpflicht',
-  },
-  {
-    id: 4,
-    name: '5. PK',
-    path: '/student/pruefungskomponente',
-  },
-  {
-    id: 5,
-    name: 'Einstellungen',
-    path: '/settings',
-  },
-];
-
-function Toolbar() {
+function Toolbar(props) {
+  const { isLoggedIn, role, accessibleComponents } = props;
```

Comment:
kannst du die props noch direkt in die Parameter schreiben? 
also:
`function Toolbar ({ isLoggedIn, role , .. })`

Reply:
Jan sagt Geschmackssache ob ich es so lassen, also lass ich es so :D